libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -23,4 +23,5 @@ class AP_BattMonitor_SMBus : public AP_BattMonitor_Backend
 
 // include specific implementations
 #include "AP_BattMonitor_SMBus_PX4.h"
-#include "AP_BattMonitor_SMBus_I2C.h"
+#include "AP_BattMonitor_SMBus_Solo.h"
+#include "AP_BattMonitor_SMBus_Maxcell.h"
```

Comment:
I think that it is unnecessary if it can distinguish by the value of I2C registers.